packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -177,7 +176,19 @@ export const useCCTPDeposits = ({
         l1ChainId: _l1ChainId,
         pageNumber: _pageNumber,
         pageSize: _pageSize
-      }).then(deposits => parseSWRResponse(deposits, _l1ChainId))
+      })
+        .then(deposits => parseSWRResponse(deposits, _l1ChainId))
+        .then(deposits => {
+          return {
```

Comment:
Before we showed 'Calculating' for CCTP statuses after showing the tx.

We can't do it now because we need to show them in a specific tab (settled or pending). We have to make a few changes to calculate this status before showing the tx.